ext.php

```diff
@@ -20,8 +20,8 @@
 */
 class ext extends \phpbb\extension\base
 {
-	/** @var string Require phpBB 3.1.3 due to the use of new exception classes */
-	const PHPBB_MIN_VERSION = '3.1.3';
+	/** @var string Require phpBB 3.1.5 due to the use of new template events */
+	const PHPBB_MIN_VERSION = '3.1.4';
```

Comment:
We could but I literally have a PR lined up switching this all back to 3.1.5 as it should be, using `>=` too. This is just because we can't develop without these changes on the latest branch right now.